spring-cloud-sleuth-core/src/main/java/org/springframework/cloud/sleuth/autoconfig/TraceAutoConfiguration.java

```diff
@@ -33,7 +33,12 @@
 import org.springframework.context.annotation.Configuration;
 
 /**
+ * {@link org.springframework.boot.autoconfigure.EnableAutoConfiguration Auto-configuration}
+ * to enable tracing via Spring Cloud Sleuth. Sets up basic building blocks for tracing.
```

Comment:
`Sets up basic building blocks for tracing.` doesn't really say anything.. I'd delete it